src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java

```diff
@@ -790,7 +790,8 @@ public void visitTerminal(Terminal terminal) {
 
     private static void checkBranch(Network network, String branchId) {
         Branch<?> branch = network.getBranch(branchId);
-        if (branch == null) {
+        DanglingLine danglingLine = network.getDanglingLine(branchId);
```

Comment:
Check branch then dangling line to avoid getting both all the times

Reply:
I cannot because I will fall in the exception whereas I have a dangling line.

Reply:
Done